proof/crefine/AARCH64/Recycle_C.thy

```diff
@@ -1068,8 +1062,21 @@ lemma cancelBadgedSends_ccorres:
    apply (clarsimp split: if_split)
    apply (drule sym_refsD, clarsimp)
    apply (drule(1) bspec)+
-   by (auto simp: obj_at'_def projectKOs state_refs_of'_def pred_tcb_at'_def tcb_bound_refs'_def
-              dest!: symreftype_inverse')
+apply (frule ksReadyQueues_asrt_ksReadyQueues_head_end)
+apply (frule invs_pspace_aligned')
+apply (frule invs_pspace_distinct')
+apply (frule (2) ksReadyQueues_asrt_ksReadyQueues_head_end_tcb_at')
+
+apply (auto simp: obj_at'_def projectKOs state_refs_of'_def pred_tcb_at'_def tcb_bound_refs'_def
+
+              dest!: symreftype_inverse')[1]
+apply (frule ksReadyQueues_asrt_ksReadyQueues_head_end)
+apply (frule invs_pspace_aligned')
+apply (frule invs_pspace_distinct')
+apply (frule (2) ksReadyQueues_asrt_ksReadyQueues_head_end_tcb_at')
+
+apply safe
```

Comment:
something is off in this part, it's either indentation or something is being applied after `auto`

Reply:
Definitely looks off, but might only be the indent.